### PR TITLE
Initial files

### PR DESCRIPTION
**Pull request by bayan-404**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (yaqootturman) [inline public/index.html:1]: Delete Comments

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (yaqootturman) [inline src/App.css]: It's an empty so delete it and delete the related sentence in App.js

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 2 above)
Comment (bayan-404) [reply src/App.css]: done :octocat:

File: public/index.html (hunk 1 above)
Comment (bayan-404) [reply public/index.html:1]: done :octocat:

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
```
Comment (yaqootturman) [inline public/index.html]: Okey, there is no manifest file anymore... so why not deleted?

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (yaqootturman) [inline public/index.html]: there is no logo!

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>  
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```
Comment (yaqootturman) [inline public/index.html]: You don't have a favicon, you can add a special one for our site

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (yaqootturman) [inline public/index.html]: Change the description

File: public/index.html [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <title>React App</title>
```
Comment (yaqootturman) [inline public/index.html]: change this too!

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: public/index.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="an app for Jokes"
+    />
+    <title>Jokes</title>
```
Comment (yaqootturman) [inline public/index.html]: The title is not Jokes and please add suitable favicon

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 8 above)
Comment (bayan-404) [reply public/index.html]: done .

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**